### PR TITLE
Filtering builders by tags - removed second box - extended sets of fi…

### PR DESCRIPTION
**Pull request by kkateq**
…lters with end conditions has been added

### CODE REVIEW

File: www/script/project/rtBuilders.js [hunk 1]
```diff
@@ -157,7 +120,8 @@ define(function (require) {
 
             tags.clear();
             $.each(data, function eachBuilder(i, builder) {
-                tags = tags.add(rtBuilders.formatTags(builder.tags, branch_type));
+                var builderTags = rtBuilders.formatTags(builder.tags, branch_type);
+                tags = tags.add(builderTags);
```
Comment (opdude) [inline www/script/project/rtBuilders.js]: Is this really doing the right thing? I would assume just tags.add(builderTags) is the right thing?

File: www/script/project/rtBuilders.js [hunk 2]
```diff
@@ -165,19 +129,23 @@ define(function (require) {
                         branch_tags.add(tag.toLowerCase());
                     }
                 });
+
+                if (builderTags.length > 1){
+                    tags.add(builderTags.join(tagSeparator));
```
Comment (opdude) [inline www/script/project/rtBuilders.js]: Didn't we already do this above?

File: www/script/project/rtBuilders.js (hunk 1 above)
Comment (kkateq) [inline www/script/project/rtBuilders.js]: Right, fixed

File: www/script/project/rtBuilders.js (hunk 2 above)
Comment (kkateq) [inline www/script/project/rtBuilders.js]: No, we don't

Comment (opdude) [inline www/script/project/rtBuilders.js]: So we'll have the individual tags and then finally an entry we each comma seperated, whats the reasoning behind that need?

Comment (kkateq) [inline www/script/project/rtBuilders.js]: So this is an alternative implementation of AND condition, now list of tags contains not only unique tag lists - but unique existing combinations.

**Pull request merged**